sdk/cognitiveservices/Azure.AI.Speech.Transcription/samples/Sample_3_TranscribeWithOptions.md

```diff
@@ -0,0 +1,141 @@
+    {
+        Console.WriteLine($"{phrase.Offset}-{phrase.Offset+phrase.Duration}: {phrase.Text}");
+    }
+}
+```
+
+## Transcribe with Profanity Filter Options
+
+To transcribe a file using profanity filters, create a stream from the file, specify the filter mode in the `TranscriptionOptions` and call `TranscribeAsync` on the `TranscriptionClient` clientlet. This method returns the transcribed phrases and total duration of the file.
+
+```C# Snippet:TranscribeWithProfinityFilter
+using (FileStream fileStream = File.Open(filePath, FileMode.Open))
+{
+    var options = new TranscriptionOptions();
+    options.ProfanityFilterMode = ProfanityFilterMode.Masked;
+
+    var request = new TranscribeRequest(fileStream);
+    request.options = options;
+
+    var response = await client.TranscribeAsync(request);
+
+    Console.WriteLine($"File Duration: {response.Value.Duration}");
+    foreach (var phrase in response.Value.PhrasesByChannel.First().Phrases)
+    {
+        Console.WriteLine($"{phrase.Offset}-{phrase.Offset+phrase.Duration}: {phrase.Text}");
+    }
+}
+```
+
+## Transcribe with Active Channels Options
+
+To transcribe a file using only a subset of the channels, create a stream from the file, specify the 0-based indices of the active channels in the `TranscriptionOptions` and call `TranscribeAsync` on the `TranscriptionClient` clientlet. This method returns the transcribed phrases and total duration of the file.
+
+If not specified, multiple channels are merged and transcribed jointly. Only up to two channels are supported.
+
+```C# Snippet:TranscribeWithActiveChannels
+using (FileStream fileStream = File.Open(filePath, FileMode.Open))
+{
+    var options = new TranscriptionOptions();
+    options.ActiveChannels.Add(0);
+
+    var request = new TranscribeRequest(fileStream);
+    request.options = options;
+
+    var response = await client.TranscribeAsync(request);
+
+    Console.WriteLine($"File Duration: {response.Value.Duration}");
+    foreach (var phrase in response.Value.PhrasesByChannel.First(p => p.Channel == 1)().Phrases)
+    {
+        Console.WriteLine($"{phrase.Offset}-{phrase.Offset+phrase.Duration}: {phrase.Text}");
+    }
+}
+```
+
+## Transcribe with Diarization Options
+
+To transcribe a file with speaker identification, create a stream from the file, specify the diarization options in the `TranscriptionOptions` and call `TranscribeAsync` on the `TranscriptionClient` clientlet. This method returns the transcribed phrases and total duration of the file.
+
+If not specified, no speaker information is included in the transcribed phrases.
+
+```C# Snippet:TranscribeWithDiarization
+using (FileStream fileStream = File.Open(filePath, FileMode.Open))
+{
+    var diarizationOptions = new TranscriptionDiarizationOptions();
+    diarizationOptions.Enabled = true;
+    diarizationOptions.MaxSpeakers = 2;
+
+    var options = new TranscriptionOptions();
+    options.DiarizationOptions = diarizationOptions;
+
```

Comment:
Is the newer initialization syntax more friendly?

var options = new TranscriptionOptions()
{
  DiazirationOptions = new()
  {
    Enabled= true;
    MaxSpeakers = 2
  }
}